pkg/trace/trace.go

```diff
@@ -16,21 +16,13 @@ package trace
 
 import (
 	"fmt"
-	"os"
 	"runtime"
 	"time"
 
-	log "github.com/Sirupsen/logrus"
+	"github.com/Sirupsen/logrus"
 )
 
-var Logger = &log.Logger{
-	Out: os.Stderr,
-	// Apparently this forces terminal handling even when Out isn't a TTY (like
-	// on a VCH, where STDOUT/STDERR are currently redirected to a file).
-	Formatter: &log.TextFormatter{ForceColors: true},
-	Hooks:     make(log.LevelHooks),
-	Level:     log.InfoLevel,
-}
+var Logger = logrus.New()
```

Comment:
@hmahmood did the unit test pass on drone?  It would fail in very much the same way on the VCH.